addons/im_livechat/views/im_livechat_channel_views.xml

```diff
@@ -139,6 +139,14 @@
                                         </div>
                                     </group>
                                 </group>
+                                <group class="row-gap-0">
+                                    <field name="concurrent_session" string="Concurrent sessions" help="If limited, operators will only handle the selected number of sessions at a time. While on a call, they will not receive new conversations." widget="radio" options="{'horizontal': true}"/>
+                                    <div invisible="concurrent_session != 'limited'" class=""/>
```

Comment:
@alexkuhn Do we need this extra `<div>`?

Reply:
It's probably creating the empty left column.